.travis.yml

```diff
@@ -68,6 +68,10 @@ dist: trusty
 script:
   - make && sudo make install && make installcheck
 
+after_failure:
+  - cat regression.diffs || true
+  - cat regression.out || true
+
 after_script:
```

Comment:
you'll note that it does the output by default here, I'd rather keep it outputting regardless, since that includes "failure".